src/index.html

```diff
@@ -19,9 +19,423 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
```

Comment:
Same issue here. You are linking a SCSS file in your HTML. You should compile your SCSS to CSS and then link the CSS file.